Makefile

```diff
@@ -297,6 +297,16 @@ $(OBJ_PATH)/%.o: $(SRC_PATH)/%.c
 	@echo -n "compile "
 	@$(BUILD_CMD)
 
+FMT_FILES += $(THEMIS_FMT)
+FMT_FILES += $(SOTER_FMT)
+
+$(OBJ_PATH)/%.fmt: CMD = CFLAGS='$(CFLAGS)' ./scripts/fmt.sh $< $@
```

Comment:
- I personally don't favour hiding the code in a separate script (unless it is too complicated), because it just creates another level of indirection
- let's keep the language suffix, so we know which files are being formatted (`.c` for formatting C code). Later we should reuse the framework to format other languages (`.go` and `.rs` at least).

Reply:
I put it into a script because we'll probably have to write almost the same Makefile targets in what seems like 5+ places (JNI wrapper, Soter and Themis tests, ThemisPP code and tests, JSThemis code).

While the C compiler is easy to use via `$(CC)`, this formatting abomination is not. And it has two operation modes with different command lines. So I used a script to avoid duplicating the code.

The same logic applies to suffixes. We'll have to format at least *.c, *.h, *.hpp, *.cpp files. I would not like to write the same targets four times. The script could guess the language by the file extension and run an appropriate formatter.

Reply:
- you can write them in one place and just `include whatever.mk` where needed
- you can have a single command for multiple extensions, if they are processed the same:
```
%.foo %.bar:
    <some common processing for both *.foo and *.bar>

%.baz:
    <some other command for baz>
```